src/pmdas/denki/denki.c

```diff
@@ -448,20 +488,23 @@ denki_fetchCallBack(pmdaMetric *mdesc, unsigned int inst, pmAtomValue *atom)
 
 	if (inst != PM_IN_NULL &&
 		!((cluster == 0 && item == 0) || 
-		(cluster == 0 && item == 1)))
+	 	  (cluster == 0 && item == 1) ||
+	 	  (cluster == 1 && item == 0) ||
+	 	  (cluster == 1 && item == 1) ||
+	 	  (cluster == 1 && item == 2)))
```

Comment:
I believe this could be expressed more compactly as:
```
        if (inst != PM_IN_NULL && mdesc->m_desc.indom == PM_INDOM_NULL)
                return PM_ERR_INST;
```

and this way if new metrics are added in future we don't need to keep updating the instance-checking logic here.